presto-benchmark/src/main/java/com/facebook/presto/benchmark/SqlMultipleApproxPercentileBenchmarks.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark;
+
+import com.facebook.presto.testing.LocalQueryRunner;
+import com.google.common.collect.ImmutableMap;
+
+import static com.facebook.presto.benchmark.BenchmarkQueryRunner.createLocalQueryRunner;
+
+public final class SqlMultipleApproxPercentileBenchmarks
```

Comment:
Nice! We also generally add before and after benchmark numbers in the commit message! like here: 

https://github.com/prestodb/presto/pull/18028

So please do that.

Reply:
Great numbers!

Reply:
Added in description.